data_steward/utils/bq.py

```diff
@@ -57,3 +60,39 @@ def delete_dataset(project_id,
                           delete_contents=delete_contents,
                           not_found_ok=not_found_ok)
     logging.info(f'Deleted dataset {project_id}.{dataset_id}')
+
+
+def is_validation_dataset_id(dataset_id):
+    """
+    Check if  bq_consts.VALIDATION_PREFIX is in the dataset_id
+    :param dataset_id: 
+    :return: a bool indicating whether dataset is a validation_dataset
+    """
+    compiled_exp = re.compile(bq_consts.VALIDATION_DATASET_REGEX)
+    return compiled_exp.match(dataset_id)
+
+
+def get_latest_validation_dataset_id(project_id):
+    """
+    Get the latest validation_dataset_id based on most recent creationTime. 
+    :param project_id: 
+    :return: the most recent validatioN_dataset_id
+    """
+
+    pattern = re.compile(bq_consts.VALIDATION_DATASET_REGEX)
+    dataset_id = os.environ.get(bq_consts.MATCH_DATASET, bq_consts.BLANK)
+    if dataset_id == bq_consts.BLANK:
+        validation_datasets = []
+        for dataset in list_datasets(project_id):
+            dataset_id = dataset.dataset_id
+            match = pattern.match(dataset_id)
+            if match:
+                validation_creation_date = datetime.strptime(
+                    match.group(1), bq_consts.VALIDATION_DATE_FORMAT)
+                validation_datasets.append(
+                    (validation_creation_date, dataset_id))
```

Comment:
It looks like we will change the naming convention to exclude dates and use only the CDR release tag in the name. Can you update this to use `dataset.created` instead?